site/content/extend/plugins/webapp/reference.md

```diff
@@ -53,6 +54,10 @@ class PluginClass {
             'custom_somepost',
             SomePost,
         );
+        registry.registerPostTypeComponent(
```

Comment:
Can we modify the server-side of this component to exercise this custom post type?

Reply:
Sounds good, and if I understand you correctly, I went to look for the server side example in documentation (first looked [server reference](https://developers.mattermost.com/extend/plugins/server/reference/), however it didn't seem appropriate.   Should it be an example at the top of the [GoDoc](https://godoc.org/github.com/mattermost/mattermost-server/plugin)?  Apologies if I have misunderstood.  (also, for QA & demo plugin possible testing there is the [/ephemeral_override](https://github.com/mattermost/mattermost-plugin-demo/pull/40))

Reply:
Please disregard my comment. It seems I reviewed this code thinking it was a plugin, but it's obvious to me now we're in the documentation block. No idea what I was thinking! :D

That being said, I think I agree with @marianunez that this example might not be necessary /here/ -- naturally, we don't demonstrate all the possible calls in this place, but defer to the demo plugin.

@scottleedavis, thoughts?

Reply:
(To complete the chain, I think https://github.com/mattermost/mattermost-webapp/pull/3024 was basically sufficient.)

Reply:
@lieut-data I agree with both of you that this example is not needed.  Thank you for your feedback and time.